infra/gp-grafana-instance/templates/01-grafana-crd.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  name: grafana-hogarama
+spec:
+  ingress:
+    enabled: true
```

Comment:
kann man in der CRD den host gleich setzen?

Reply:
Falls ja, dann grafana.apps.clustername.gepaplexx.com als variable.

Reply:
Ja, `hostname` kann man setzen. Zusätzlich eine ganze Reihe anderer Eigenschaften (`termination, annotations, path, tlsEnabled, ingressClassName, targetPort, tlsSecretName, pathType, labels`). Ich würde die erst einbauen, wenn wir sie auch wirklich benötigen um nicht unnötige Komplexität zu erzeugen.

Reply:
hostname ist meiner meinung nach ein valider punkt, der das ganze einfacher macht. ich setz den zumindest bei jeder route/jedem ingress. dann braucht man auch nicht alle links auf einer übersichtsseite haben für X cluster, sondern merkt sich einfach ein simples schema ;) 
termination und tlsenabled ist meiner meinung nach genauso wichtig. sonst exposed du schlimmstenfalls über http - würd ich zumindest testen und wenn default nicht https/tls encrypted ist ist das definitiv wichtig zu setzen.

Reply:
Ich tippe nie URLs außer google.at wenn duckduckgo zu wenig Ergebnisse bringt. :-) Aber ich verstehe die Nützlichkeit von Konventionen (Convention over Configuration). Ich habe dafür den Issue https://gepardec.atlassian.net/browse/GPX-428 angelegt.

Reply:
Von mir aus okay die generierten routen zu verwenden.